content/docs/reference/pkg/nodejs/pulumi/vault/lDAP/_index.md

```diff
@@ -1,1282 +0,0 @@
----
```

Comment:
I assume this is getting deleted bashed we changed the capitalization of its name or something?  I hope we are not actually deleting any content?

Reply:
Yeah, this is fallout from a capitalization change in the provider. https://github.com/pulumi/docs/pull/2224 cleans it up.